docs/structure.md

```diff
@@ -1,19 +1,17 @@
-# Orb Structure
+# Orb structure
 
-Each Orb is packaged as a single file. Each Orb also has its 
-own namespace to use when invoking its elements. Orbs are authored with similar 
-YAML syntax to the base CircleCI build configuration.
+Each Orb is packaged as a single file. Each Orb also has its own namespace to use when invoking its elements. Orbs are authored with similar YAML syntax to the base CircleCI build configuration.
 
 ## Anatomy of an Orb
-Orbs are composed of one or more of the following elements, each of which represents a type of invocable element in CircleCI project configuration.:
+Orbs are composed of one or more of the following elements, each of which represents a type of invocable element in CircleCI project configuration:
 
 * [commands](commands.md)
 * [jobs](jobs.md)
 * [executors](executors.md)
 
 An example orb:
 ```
-# .circleci/orbs/foo/orb.yml
+# foo Orb
```

Comment:
How _are_ we actually naming and versioning orbs? Where is that getting defined?

Reply:
The name and version should be defined outside of the actual contents of an orb, so that forking and changing them is easier. They would be fined on `publish` afaik, e.g. `circleci-cli orb publish myorb v0.1.25`.

For inline orbs, as in local in folder or nested inside config, the name is provided as a key under the `orbs` stanza. In that case, I think version is irrelevant.